source/style-guide/style/lists.txt

```diff
@@ -39,8 +39,11 @@ guidelines when introducing lists.
        period instead of a colon.
 
        .. note::
-          Avoid using fragments to introduce lists. Fragments can be
-          harder to understand than sentences.
+
+          Where possible, avoid using fragments to introduce lists. 
+          However, you may use stem sentences to avoid especially 
+          awkward or redundant constructions. Use your best discretion. 
+          Consider accessibility and screen readers.
```

Comment:
Question:

I wasn't able to find any stand-out considerations in a quick search on screen readers. Could this sentence explain or link to some of the ones that should be kept in mind?